sysl2/sysl/sysl.go

```diff
@@ -22,6 +22,12 @@ func main3(stdout, stderr io.Writer, args []string) error {
 	case "syslgen":
 		DoGenerateCode(stdout, stderr, flags, args)
 		return nil
+	case "sd":
+		err := DoGenerateSequenceDiagrams(stdout, stderr, flags, args)
```

Comment:
Why not the same as syslgen?

Reply:
Misunderstand about the console output when exception. Keep the same form with syslgen.